### ISSUE TITLE
Custom SSH port

### ISSUE THREAD

**Issue by craigfrancis** [mozilla/http-observatory-website#126]
Before running the SSH Observatory test, would it be possible to change the port number that is scanned... use 22 by default, but allow someone to change it.

And just to confirm, a non-standard port number does not help with security, it just keeps the access logs a bit clearer, due to the simple bots that keep trying (and failing) to login.

**Comment by craigfrancis** [comment]
Actually, this might not be possible... as while the "sshscan.rubidus.com" API accepts a port number via a POST parameter:

https://github.com/mozilla/ssh_scan_api/wiki/ssh_scan-Web-API

It returns `{"error":"invalid port"}` when you use something other than port 22:

https://github.com/mozilla/ssh_scan_api/blob/master/lib/ssh_scan_api/api.rb#L113
https://github.com/mozilla/ssh_scan_api/blob/master/config/api/config.yml#L9

**Comment by floatingatoll** [comment]
The public Observatory website, per standing policy, will only scan the
default ports - 22 (and 80/443). If you need to scan custom ports, you'll
need to run your own scanner instance with the restriction removed.

**Comment by craigfrancis** [comment]
Fair enough, it might be worth noting something like that on the HTTP Observatory website, but not essential; e.g. changing:

    "SSH Observatory scans are not performed automatically. Click here to initiate a scan."

    "SSH Observatory scans are not performed automatically. Click here to initiate a scan on port 22."

That said, removing the use of "click here" would be good ([details](https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/)), so maybe:

    "Initiate a scan on port 22 - SSH Observatory scans are not performed automatically"

**Comment by dennis-fernuni** [comment]
Maybe it is possibe to do a srv lookup to determine the ssh port of a domain:
host -t SRV _ssh._tcp.example.com
dig _ssh._tcp.example.com SRV
https://en.wikipedia.org/wiki/SRV_record

This allows a custom ssh port...

**Comment by zachaysan** [comment]
I also would appreciate the option of non-standard port scanning.
